content/en/bits_ai/bits_ai_sre/investigate_alerts.md

```diff
@@ -43,30 +43,23 @@ An investigation initiates when a monitor transitions to the alert state. Transi
 
 ### Monitor requirements for Bits AI SRE
 
-Bits is able to run investigations on monitors that fulfill all three of the following requirements:
-1. **Monitor Type**: The monitor must be a metric, logs, APM (`APM Metrics` type only), anomaly, forecast, integration, or outlier monitor.
+Bits is able to run investigations on the following monitor types:
+  - metric
+  - anomaly
+  - forecast
+  - integration
+  - outlier
+  - logs
+  - APM (`APM Metrics` type only)
 
-2. **Service scope**: The monitor must have one of the following:
-   - A monitor query filtered by a service
-   - A monitor query grouped by service
-   - A service tag on the monitor
+There are several things you can do to ensure your Bits AI SRE investigations are as accurate as possible.
 
-3. **Telemetry links**: For metric, anomaly, forecast, integration, and outlier monitors, the monitor message must include at least one helpful link:
-   - A Datadog dashboard
-   - A logs query
-   - A trace query
-   - A Datadog notebook with helpful widgets
-   - A [Confluence page](#configure-knowledge-base-integrations) 
+First, make sure there is **at least one helpful telemetry link** inside of the monitor message. This link could be a Datadog dashboard, logs query, trace query, a Datadog notebook with helpful widgets, or a [Confluence page](#configure-knowledge-base-integrations). Think of the first page you'd navigate to in Datadog if this monitor were to fire.
```

Comment:
```suggestion
**First**, make sure there is **at least one helpful telemetry link** inside of the monitor message. This link could be a Datadog dashboard, logs query, trace query, a Datadog notebook with helpful widgets, or a [Confluence page](#configure-knowledge-base-integrations). Think of the first page you'd navigate to in Datadog if this monitor were to be triggered.
```

Reply:
updated in code since I couldn't directly apply after Kai's edits!